test/cql-pytest/test_virtual_tables.py

```diff
@@ -78,10 +78,10 @@ def test_system_config_read(scylla_only, cql):
     assert obj[0] and obj[0].isascii() and obj[0].isprintable()
     assert not obj[0].isnumeric()  # issue #11003
     # Check formatting of tri_mode_restriction like
-    # restrict_replication_simplestrategy. These need to be one of
+    # restrict_dtcs. These need to be one of
     # allowed string values 0, 1, true, false or warn - but in particular
     # non-empty and printable ASCII, not garbage.
-    assert 'restrict_replication_simplestrategy' in values
-    obj = json.loads(values['restrict_replication_simplestrategy'])
+    assert 'restrict_dtcs' in values
```

Comment:
Why do you check pre-existing `restrict_dtcs` here but not the newly added `replication_strategy_warn_list`/`replication_strategy_fail_list`?

Reply:
As the above comment explains:

> Check formatting of tri_mode_restriction like

`restrict_dtcs` is a tri-mode config option, and because we want to get rid of `restrict_replication_simplestrategy` (which I initially did, but reverted the change), I needed to replace it with another such option.

Reply:
Ahh, but maybe you want me to also add a check if my new config values are parsed correctly? Hm, I could, though it's checked in cql-pytest I provide already.

Reply:
No, no, I just noticed that `restrict_dtcs` doesn't appear anywhere else in this patch and was surprised by that. Then I checked that it's some other pre-existing option and was surprised for the second time. But now I see the motivation and it's correct.